lib/ProductOpener/Attributes.pm

```diff
@@ -813,7 +813,7 @@ sub compute_attribute_nova($$) {
 			$match = 100;
 		}
 		elsif ($nova_group == 3) {
-			$match = 50;
+			$match = 75;
 		}
```

Comment:
Optional: This part could be better written as a lookup in a hash with scores.

Reply:
ok